### ISSUE TITLE
"stable" keyword

### ISSUE THREAD

**Issue by JaKXz** [jasongin/nvs#120]
First of all, this is awesome, thank you for creating it! 🎉 

`nvm` has a `stable` keyword that points to the latest stable release of node (that's ahead of the `lts/*` release). It would be great to have that in `nvs` so I don't have build failures like this: https://travis-ci.org/istanbuljs/nyc/builds/476649312?utm_medium=notification&utm_source=email

**Comment by JaKXz** [comment]
Upon further reading... this is not really a great issue. It would be _nice to have_ parity with `nvm` - but that might be asking too much so feel free to close this issue. Thanks again!

**Comment by ljharb** [comment]
I’d discourage you from using “stable” with nvm anyways, since everything is stable. Use `node` or `lts/*` to indicate which “latest” you want.

**Comment by TheJaredWilcurt** [comment]
Agreed that `nvs` should not match `nvm` for parity sake. For this case I think copying Volta's approach is the most sensible.

* `volta pin node@latest` - Finds the latest stable release (v25.2.1), pins it locally, downloads it (if needed), switches to it.
* `volta pin node@lts` - Finds the latest long-term-support release (v24.12.0), pins it locally, downloads it (if needed), switches to it.

Where `lts` means the latest stable long-term-support release, and `latest` means the latest stable release, which may or may not be a long-term-support release.
